docs/tutorials/writing-views.md

```diff
@@ -193,7 +193,7 @@ REQ also helps us with request variable validation. For example:
   not automatically marshall the input from JSON).
 
 - Since there is no need to JSON-encode strings, usually simply
-  `my_string=REQ()` is correct. One can pass e.g.
+  `my_string=REQ()` is correct. One can pass e.g.,
```

Comment:
This one might be invalid; "one can pass, for example," would be nicer.

Just gonna merge since this isn't a regression.

Reply:
I suppose this is a little list to look through:

```
$ git grep '[a-z] e.g., '
docs/documentation/overview.md:(so you can also use e.g., `firefox docs/_build/html/index.html` from
docs/production/authentication-methods.md:reject authentication requests e.g., based on IP address of the
docs/subsystems/events-system.md:list of user IDs correctly. There can be security problems if e.g., an
docs/subsystems/hashchange-system.md:  several UI-manipulating things including e.g., loading the channels
docs/subsystems/markdown.md:processor object via e.g., `_md_engine.zulip_db_data`, and then
docs/subsystems/notifications.md:    where e.g., the user closes their last Zulip tab and we get the
docs/subsystems/pointer.md:When you unnarrow using e.g., the `a` key, you will automatically be
docs/subsystems/presence.md:- We provide the data for e.g., whether the user was online on their
docs/subsystems/queuing.md:  queue processor manually using e.g., `./manage.py process_queue --queue=user_activity`.
docs/subsystems/realms.md:These records are also useful if you want to e.g., run the Puppeteer tests
docs/subsystems/schema-migrations.md:  be done in batches of e.g., 1000 objects (e.g., a `Message` or
templates/zerver/development/dev_tools.html:                <li><code>./manage.py populate_db</code>: Rebuilds database. Has options to e.g., create 3K users for testing.</li>
tools/droplets/README.md:   - Name it e.g., `base-ubuntu-22-04.zulipdev.org`
web/styles/portico/comparison_table.css:           colliding with the left screen edge, though. */
```